xla/hlo/transforms/collectives/collective_quantizer.cc

```diff
@@ -36,13 +37,29 @@ struct ConversionSubgraph {
   // Unary instructions following a dequantization or preceding a quantization
   // in top-down order (operand-to-user).
   std::vector<HloInstruction*> unaries;
+
+  std::string ToString() const {
+    auto short_string = [](const HloInstruction* instr) -> std::string {
+      return instr ? instr->ToShortString() : "null";
+    };
+
+    std::ostringstream oss;
+    oss << " conversion: " << short_string(convert)
```

Comment:
Don't start with a space. And change users of this method to take into account there isn't a space.